snap/snapenv/snapenv_test.go

```diff
@@ -108,14 +110,31 @@ func (ts *HTestSuite) TestUser(c *C) {
 }
 
 func (ts *HTestSuite) TestUserForClassicConfinement(c *C) {
+	dirs.SetRootDir(c.MkDir())
+	defer dirs.SetRootDir("/")
+	c.Assert(os.MkdirAll(dirs.FeaturesDir, 0755), IsNil)
+
+	// With the classic-preserves-xdg-runtime-dir feature disabled the snap
+	// per-user environment contains an override for XDG_RUNTIME_DIR.
 	env := userEnv(mockClassicSnapInfo, "/root")
+	c.Assert(env, DeepEquals, map[string]string{
+		// NOTE: Both HOME and XDG_RUMTIME_DIR are not defined here.
```

Comment:
```suggestion
		// NOTE: Both HOME and XDG_RUNTIME_DIR are not defined here.
```

:thinking: still not rumtime